### DISCUSSION TITLE
Problema ao usar OctetStream

### DISCUSSION THREAD

**Discussion by osmargdi** [HashLoad/horse#377] [General]
Quando eu uso THorse.Use(OctetStream) e crio uma rota Post da erro interno no servidor
   THorse.Use(OctetStream);

   THorse.Post('/teste',
      procedure(Req: THorseRequest; Res: THorseResponse)
      begin
         Res.Send('teste');
      end);

Retorno
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Error 500: Internal Server Error</title>
</head>

o que estou fazendo de errado?

**Comment by viniciussanchez** [comment]
Anexe um exemplo por favor, gerando o erro... aparentemente olhando assim não era para dar problema. Quais as versões você está usando?

**Reply by osmargdi** [reply]
"github.com/hashload/horse": "^3.1.5",
	"github.com/hashload/horse-octet-stream": "^1.9.10",
	
não consegui anexar arquivo .dpr entao renomiei para .txt OK.

[teste.txt](https://github.com/HashLoad/horse/files/14112694/teste.txt)

**Reply by osmargdi** [reply]
![image](https://github.com/HashLoad/horse/assets/17091288/f05c8dc8-9e6f-44cb-bc71-2a74819e237d)

**Comment by osmargdi** [comment]
"github.com/hashload/horse-octet-stream": "^1.9.11",
Atualizei para essa versao e funcionou, vlw.
